### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by ShubhamPandey28**
I have replaced `select` with `set_config` as discussed in issue #69 , I have also added some presentation resources from LPC2019 and ELCE2019

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 __pycache__/
+.ipynb_checkpoints/
```
Comment (rralf) [inline .gitignore:2]: This change is irrelevant. It should move to your global gitignore.

**Comment by rralf** [comment]
Hi,

(+Cc @bulwahn)

Will accept the README patch but drop the changes to gitignore, and squash the other two patches. Thanks!

**Comment by rralf** [comment]
Thanks, rebased an merged to the next branch.

File: .gitignore (hunk 1 above)
Comment (ShubhamPandey28) [reply .gitignore:2]: > Thanks, rebased an merged to the next branch.

Thanks

**Pull request closed without merging**